pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,111 @@
+# /spec/features would be owned by the test-analytics team.
+
+/specs/ test-analytics
+/specs/features
+```
+
+### Example TESTOWNER file
+
+```bash
+# This is a comment.
+# Each line is a file pattern followed by one or more team slugs.
+
+# These teams will be the owners for everything in
+# the repo. While both teams own the test, the first team listed will
+# become the default owner for tests in this directory. Unless
+# a later match takes precedence, team-slug-1 will have any
+# flaky tests assigned to them by default.
+*       team-slug-1 team-slug-2
+
+# Order is important; the last matching pattern takes the most
+# precedence. Any test file ending with _spec.rb will be assigned
+# to the test-analytics team and not team-slug-1.
+*_spec.rb    test-analytics #This is an inline comment.
+
+# Only Buildkite teams can be specified as test owners. Teams should
+# be identified by their slug. Teams must have
+# explicit access to the suite the test belongs to. In this example,
+# the pipelines team owns all .rb files.
+*.rb pipelines
+
+# In this example, packages owns any files in the spec/packages/
+# directory at the root of the test directory and any of its
+# subdirectories.
+/spec/packages/ packages
+
+# The `spec/features/*` pattern will match files like
+# `spec/features/application_spec.rb` but not further nested files like
+# `spec/features/pipelines/application_spec.rb`.
+spec/features/*  test-analytics
+
+# In this example, pipelines owns any file in a pipelines directory
+# anywhere in your test directory.
+pipelines/ pipelines
+
+# In this example, packages owns any file in the `/spec/packages`
+# directory in the root of your test directory and any of its
+# subdirectories.
+/spec/packages/ packages
+
+# In this example, test-analytics owns any test in a `/analytics` directory such as
+# `/models/analytics`, `/features/analytics`, and `/models/organizations/analytics`. # Any tests in an `/analytics` directory will belong to team test-analytics.
+**/analytics test-analytics
+
+# In this example, pipelines owns any test in the `/spec`
+# directory in the root of your test directory except for the `/spec/models/packages`
+# subdirectory, as this subdirectory has its own owner packages
+/spec/ pipelines
+/spec/models/packages packages
+```
+{: codeblock-file="TESTOWNERS"}
+
+### Teams
+A TESTOWNER uses Buildkite team slugs instead of user names. Your team slug will be your team name in snake-case. You can view your teams in your organization settings, or fetch them from our API:
+
+- [List teams from REST API](/docs/apis/rest_api/teams)</li>
+- [List teams from Graphql API](/docs/apis/graphql/schemas/object/team)</li>
+
+> 🚧 Team permissions
+>It is important to [ensure these teams have permission to access the suite](/docs/test-analytics/permissions#manage-teams-and-permissions) the file is uploaded to, otherwise these ownership records will not be created. You can check these permissions in your organization's team settings.
```

Comment:
I had to do some mental gymnastics on this sentence, talking about permissions is damn hard. 

How about?

> Teams listed in the the TESTOWNERS must have permission to access the suite before ownership records are created.

🤷🏻 i hate talking about permissions 😭 it breaks my brain.